apps/web/components/common/Sidebar/Layout.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import React, { type ReactNode, useState } from "react";
+import Sidebar from "./index";
+
+interface LayoutProps {
+  children: ReactNode;
+}
+
+function Layout(props: LayoutProps): JSX.Element {
```

Comment:
```suggestion
export default function Layout for what? (props: LayoutProps): JSX.Element {
```
이런식으로 export하는게 우리 컨벤션에 맞는 방식인거같아...!
추가로 어떤 layout인지 조금 더 명확한 네이밍을 고민해보는게 좋을거같아